### DISCUSSION TITLE
Originally Available Field Wrong Dates

### DISCUSSION THREAD

**Discussion by deathzmage** [CodyBerenson/PGMA-Modernized#157] [Q&A]
My date field for unmatched items are always displaying the date 1969-12-31. Even after I change it to something else, when I go back into edit, it says 1969-12-31 still. How can I go about fixing this?

**Comment by JPH71** [comment]
The date 1969-12-31... is the default date field...

If it is not changing it might be because the field is looked... try and
unlock it before changing it...

I think you have to  double click on the padlock .... texting from memory
here...

Jason x

**Reply by deathzmage** [reply]
I tried that and it still reverts back to the default date. It's strange because I have been using these agents for a long time now and nothing like this has ever happened.

**Comment by JPH71** [comment]
Try creating a new library and adding some films to it and scrape... make
sure you are following the naming conventions..
